src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Metadata.php

```diff
@@ -197,4 +126,17 @@ class Metadata
      * @var string
      */
     public $organizationUrlNl;
+
+    public function setAttribute(string $property, Attribute $value)
+    {
+        $this->attributes[$property] = $value;
+    }
+
+    public function getAttribute(string $property): Attribute
+    {
+        if (array_key_exists($property, $this->attributes)) {
+            return $this->attributes[$property];
+        }
+        return new Attribute();
```

Comment:
May I suggest you return a 'NullAttribute' here. Or, more strictly speaking (see https://tuhrig.de/find-vs-get/) throw an exception here!